src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/ObjectionStatus.java

```diff
@@ -5,5 +5,10 @@
     PROCESSED,
     SUBMITTED,
     INELIGIBLE_COMPANY_STRUCK_OFF,
-    INELIGIBLE_NO_DISSOLUTION_ACTION
+    INELIGIBLE_NO_DISSOLUTION_ACTION;
+
+    public boolean isEligibilityError() {
```

Comment:
I wouldn't class it as an error as they can be ineligible without causing an error. How about isIneligibleStatus()?

Reply:
Or maybe just 'isIneligible()' (status is already present in the class name...)

Reply:
Renamed